requirements-test.txt

```diff
@@ -4,5 +4,7 @@ freezegun
 # TODO: change this when next point version of openwisp-utils is released
 openwisp-utils[qa] @ https://github.com/openwisp/openwisp-utils/tarball/master
 openwisp-controller~=0.8
+# TODO: change this when next point version of openwisp-users is released
+openwisp-users @ https://github.com/openwisp/openwisp-users/tarball/master
```

Comment:
Had to do this, as when openwisp-controller~=0.8 gets installed it degrades openwisp-user to 0.5.1(which doesn't have the `DjangoModelPermisisons` class) hence, CI tests are failing due to this.

Reply:
You could have changed openwisp-controller master, but it's ok, in any case we'll have to come back to this file to update it when the next release is ready.